apps/cli/src/services/wallet/list.ts

```diff
@@ -0,0 +1,35 @@
+import { IDatabase, IWallet } from '@cypherock/db-interfaces';
+import colors from 'colors/safe';
+
+export interface IListWalletFlags {
+  short?: boolean;
+}
+
+const walletPropertiesToDisplay = ['__id', 'name', 'hasPin', 'hasPassphrase'];
```

Comment:
```suggestion
const walletPropertiesToDisplay: (keyof IWallet)[] = [ '__id', 'name', 'hasPin', 'hasPassphrase', ];
```
Doing something like this can prevent use of any on line 13